test_pytest_timeout.py

```diff
@@ -390,15 +396,23 @@ def test_marker_help(testdir):
 
 
 @have_spawn
-def test_suppresses_timeout_when_pdb_is_entered(testdir):
-    pytest.importorskip("pexpect")
+@pytest.mark.parametrize(
+    "debugging_module,debugging_set_trace",
+    [
+        ("pdb", "set_trace"),
+        ("ipdb", "set_trace"),
+        ("pydevd", "settrace"),
```

Comment:
I had a quick play with pydevd and I don't think this can be done the same way.  It behaves very differently and needs a server to be running.  The docs for it's api seem a bit thin so I don't yet know how to do this, but pydevd.settrace() tries to connect to a server at least.  I'll need to explore this more to know how pydevd works.

Reply:
I was able to get it to connect to a server, but for some reason it didn't set a trace function from what I can tell?